terraform/deployments/govuk-publishing-infrastructure/outputs.tf

```diff
@@ -1,7 +1,3 @@
 output "shared_redis_cluster_host" {
-  value = aws_route53_record.shared_redis_cluster_internal_service_record.fqdn
-}
-
-output "shared_redis_cluster_port" {
-  value = var.shared_redis_cluster_port
```

Comment:
Why I added port before was if this output was used to construct the Redis URL for the apps, the URL has a port in it. I see that now this is hardcoded to its default/conventional port.

Reply:
Ah yeah thanks I forgot to mention in the description that I'd changed that. Simplifies things a bit (by eliminating a var that adds no real benefit) since we would never have any reason to run it on a non-standard port these days.